.github/workflows/build-staging-android.yml

```diff
@@ -52,9 +52,9 @@ jobs:
         env:
           GPG_KEYSTORE_FILE: ${{ secrets.GPG_KEYSTORE_FILE }}
           GPG_KEYSTORE_PASS: ${{ secrets.GPG_KEYSTORE_PASS }}
-      - name: Generate icons
+      - name: Generate native assets
         if: steps.apk-cache.outputs.cache-hit != 'true'
-        run: brew install imagemagick && yarn icons
+        run: brew install imagemagick && yarn generate-native-assets
```

Comment:
Hva med å slå sammen 
name: Generate native assets og name: Set environment til ett steg : sh setup.sh $APP_ENVIRONMENT $APP_ORG

Reply:
Så:
`      - name: Setup native assets and env
        if: steps.apk-cache.outputs.cache-hit != 'true'
        run: brew install imagemagick && sh setup.sh $APP_ENVIRONMENT $APP_ORG`

Reply:
Vi er nødt til å switche ut environment uavhengig av generering av assets.

Environment brukes til andre ting, men assets er unødvendig (tidskrevende) å generere hvis vi gjenbruker cachet pakke fra forrige bygg (ref. linje 56: `if: steps.apk-cache.outputs.cache-hit != 'true'`)

Så det er grunnen til at jeg har måtte dele det opp slik.

Reply:
Oki